src/legacy/core_plugins/kibana/public/dashboard/panel/panel_utils.test.js

```diff
@@ -54,9 +74,9 @@ test('convertPanelDataPre_6_3 scales panel dimensions', () => {
       x: 2,
       y: 5,
     },
-    version: '6.2.0'
+    version: '6.2.0',
   };
-  const updatedPanel = PanelUtils.convertPanelDataPre_6_3(oldPanel);
+  const updatedPanel = PanelUtils.convertPanelDataPre_6_3(oldPanel, true);
```

Comment:
This line changed functionality by making the second parameter true, I believe this is causing the current test failure.

Reply:
Amazing! thanks!